app/services/users/delete_account_service.rb

```diff
@@ -65,8 +65,7 @@ def create_audit_record
         school_id: @user.school_id,
         metadata: {
           email: @user.email,
-          account_deletion_notification_sent_at:
-            @user.account_deletion_notification_sent_at&.iso8601
+          account_deletion_notification_sent_at: @user.account_deletion_notification_sent_at&.iso8601
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)